h: No such file or directory    27 | #include "common/atoms-extern.h"

```

[ 83%] Built target generate-examples
[ 84%] Built target ldoc
[ 84%] Built target version_stamp
[ 91%] Built target man
[ 91%] Building C object CMakeFiles/test-systray.dir/tests/test-systray.c.o
In file included from /home/eric/awesome/tests/test-systray.c:29:
/home/eric/awesome/build/common/atoms.h:27:10: fatal error: common/atoms-extern.
h: No such file or directory
   27 | #include "common/atoms-extern.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[3]: *** [CMakeFiles/test-systray.dir/build.make:82: CMakeFiles/test-systray.dir/tests/test-systray.c.o] Error 1
make[2]: *** [CMakeFiles/Makefile2:11344: CMakeFiles/test-systray.dir/all] Error 2
make[1]: *** [Makefile:171: all] Error 2
make: *** [Makefile:45: package] Error 2

```
pop!_OS 20.04 (NVIDIA)

I am pretty bad at Linux, but I tried reinstalling atom which of course did nothing.
I am stuck and really want to use Awesome, because it's Awesome.

I'm having same problem in arch linux.
(AUR: https://aur.archlinux.org/packages/awesome-git/)

```
[ 89%] Built target version_stamp
[ 89%] Building C object CMakeFiles/test-systray.dir/tests/test-systray.c.o
In file included from /home/dwsong07/Downloads/awesome-git/src/awesome-git/tests/test-systray.c:29:
/home/dwsong07/Downloads/awesome-git/src/build/common/atoms.h:27:10: fatal error: common/atoms-extern.h: No such file or directory
   27 | #include "common/atoms-extern.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/test-systray.dir/build.make:76: CMakeFiles/test-systray.dir/tests/test-systray.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:10924: CMakeFiles/test-systray.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
==> ERROR: A failure occurred in build().
    Aborting...

```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -404,6 +404,8 @@ target_link_libraries(test-gravity
     ${AWESOME_COMMON_REQUIRED_LDFLAGS} ${AWESOME_REQUIRED_LDFLAGS})
 
 add_executable(test-systray tests/test-systray.c)
+add_dependencies(test-systray generated_sources)
+
 target_link_libraries(test-systray
     ${AWESOME_COMMON_REQUIRED_LDFLAGS} ${AWESOME_REQUIRED_LDFLAGS})
 

